cpp/ql/lib/semmle/code/cpp/stmts/Stmt.qll

```diff
@@ -213,6 +213,26 @@ class ConditionalStmt extends ControlStructure, TConditionalStmt { }
 class IfStmt extends ConditionalStmt, @stmt_if {
   override string getAPrimaryQlClass() { result = "IfStmt" }
 
+  /**
+   * Gets the initialization statement of this 'if' statement.
```

Comment:
This makes it clear that there might not be any initialization statement. (I know you've written it further down in the qldoc, but this phrasing matches our style guide.)
```suggestion
   * Gets the initialization statement of this 'if' statement, if any.
```